vtex/faststore
# Feat: Refactors section grid

## What's the purpose of this pull request?
This PR is a follow up from:
- https://github.com/vtex/faststore/pull/1775
- https://github.com/vtex/faststore/pull/1768

Doing these other tasks, we found that it wasn't possible to change a section from full width to grid centered, so we fixed it in order to improve the dev experience using the FS 2.0.

Additionally, we:
- Set `ProductCard` bordered as default
- Removed old `.layout__main` class
- Improved `Carousel` padding to always follow the centered grid
- Updated customization docs naming and descriptions (view @Mariana-Caetano commits after [comment](https://github.com/vtex/faststore/pull/1779#pullrequestreview-1475962544))

## How it works?

https://github.com/vtex/faststore/assets/11613011/e3f17c79-b6f9-4a06-9e60-65df0b806954

## How to test it?
All components should look the same as `main`.

`library` doc:
1. Review new doc [Customizing Pages](https://faststore-site-git-feat-section-grid-fs-1074-faststore.vercel.app/docs/customizing/pages)

`faststore/core` application:
1. Run `yarn dev`
2. Check all pages

`starter.store`:
1. Visit our [preview link](https://evergreen-git-feat-theming-test-faststore.vercel.app/)
